packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -53,6 +60,14 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 		return this._summarizeProtocolTree === true;
 	}
 
+	private _loadedGroupIdSnapshots: Record<string, ISnapshot> | undefined;
+	/**
+	 * Any snapshots that were downloaded during the initial load of the container.
```

Comment:
This comment is not entirely correct imo. This is not just initial load, but any snapshot fetched before going offline, no?

Reply:
Yeah, this comment is in the wrong place